pkg/virtctl/completion/completion.go

```diff
@@ -0,0 +1,33 @@
+package completion
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+func NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "completion SHELL",
+		Short: "Output shell completion code for the specified shell (bash or zsh)",
+		Example: `# Load the virtctl completion code for bash into the current shell
```

Comment:
Would it be worth mentioning that by convention these sorts of completion functions tend to be placed in `/etc/bash_completion.d/`? Not sure if that starts to get outside the scope of these instructions though...